.github/workflows/connector_integration_tests.yml

```diff
@@ -29,3 +29,22 @@ jobs:
         run: python ./tools/bin/ci_integration_workflow_launcher.py base-normalization source-acceptance-test source:alpha source:beta source:GA destination:alpha destination:beta destination:GA
```

Comment:
Hm, the alpha connectors are still in here, maybe you need to merge master in to this branch? Just to validate